sympy/matrices/tests/test_matrices.py

```diff
@@ -51,6 +51,10 @@ def test_sum():
     n = Matrix(1, 2, [1, 2])
     raises(ShapeError, lambda: m + n)
 
+def test_abs():
+    m = Matrix(1, 2, [-3, x])
+    n = Matrix(1, 2, [3, Abs(x)])
+    assert abs(m) == n
```

Comment:
could you add a test like this to `test_commonmatrix.py` in the `ArithmeticOnlyMatrix` section?